src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java

```diff
@@ -1154,6 +1154,50 @@ public String toString() {
     }
   }
 
+  public enum ShardStateDimension implements MetricDimension {
```

Comment:
Can we add some documentation here? For example [CacheConfigDimension](https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/master/src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java#L134) : 

```
  /*
   * column names of Cache_MaxSize table
   * cache type | sum | avg | max | min |
   *
   * <p>Example:
   * Field Data Cache|26214400.0|26214400.0|26214400.0|26214400.0
   * Shard Request Cache|80181985.0|80181985.0|80181985.0|80181985.0
   */

Reply:
Done